Content/paket.dependencies

```diff
@@ -1,7 +1,7 @@
 source https://www.nuget.org/api/v2
 
 nuget FSharp.Core  redirects:force, content:none
-nuget Fable.Core
+nuget Fable.Core content:none
```

Comment:
You should take this paket.dependencies as sample IMO

https://github.com/fable-compiler/fable-templates/blob/master/simple/Content/paket.dependencies